storage/innobase/xtrabackup/src/redo_log.cc

```diff
@@ -138,6 +139,14 @@ static bool reopen_log_files() {
   log.m_log_flags = log_flags;
   log.m_log_uuid = log_uuid;
   log.m_files = std::move(files);
+  auto file = log.m_files.find(log_get_checkpoint_lsn(log));
```

Comment:
can we also do such a check when we first open the file for checkpoint_lsn? (this code is for re-open scenario)

Reply:
code not present in the latest commit.